boards/feather_esp32/definition.json

```diff
@@ -1,12 +1,23 @@
 {
-    "boardName":"adafruit-huzzah-32",
+    "boardName":"feather_esp32",
     "mcuName":"ESP32",
     "mcuRefVoltage": 3.3,
     "VID":"0x239A",
     "PID":"0x8036",
     "displayName":"Adafruit Feather HUZZAH ESP32",
     "productPageURL":"https://www.adafruit.com/product/3405",
     "documentationURL":"https://learn.adafruit.com/adafruit-huzzah32-esp32-feather",
+    "esptool": {
+      "fileSystemSize": 94208,
+      "blockSize": 4096,
+      "offset": "0x290000",
+      "structure": {
+          "0xe000": "wippersnapper.esp32.littlefs.VERSION.boot_app0.bin",
+          "0x1000": "wippersnapper.esp32.littlefs.VERSION.bootloader.bin",
+          "0x10000": "wippersnapper.esp32.littlefs.VERSION.bin",
+          "0x8000": "wippersnapper.esp32.littlefs.VERSION.partitions.bin"
+        }
```

Comment:
All 3 boards in this PR need the "structure" property filled out with the real filenames from the firmware releases.

Note, make sure all referenced files are in the release `.zip`, I notice that `boot_app0.bin` isn't in the esp32 release zip, for example.

Reply:
Resolved in https://github.com/adafruit/Wippersnapper_Boards/pull/39/commits/31fb217f05bfffe90bd29590216045d23c7a89b2 and https://github.com/adafruit/Adafruit_Wippersnapper_Arduino/pull/227